File: app/src/main/java/basic/ex02.java
package basic;

import java.util.Locale;

public class ex02 {
    public static void main(String[] args) {
        int id = 2026001;
        double gpa = 4.18;

        System.out.printf(
                Locale.US,
                "id: %d, gpa: %.2f (cast to int: %d)%n",
                id,
                gpa,
                (int) gpa
        );
    }
}

File: app-kotlin/src/main/kotlin/basic/ex02.kt
package basic

import java.util.Locale

class ex02 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val id = 2026001
            val gpa = 4.18
            println(String.format(Locale.US, "id: %d, gpa: %.2f (cast to int: %d)", id, gpa, gpa.toInt()))
        }
    }
}
